pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/documentation/ApexDocRule.java

```diff
@@ -48,10 +48,15 @@
             booleanProperty("reportMissingDescription")
                 .desc("Report missing @description").defaultValue(true).build();
 
+    private static final PropertyDescriptor<Boolean> REPORT_MISSING_PROPERTY_DESCRIPTOR =
+            booleanProperty("reportMissingProperty")
```

Comment:
I'll rename this property to be "reportProperty" - to be more aligned with "reportPrivate" and "reportProtected".
I first thought it is similar like "reportMissingDescription" - but it is not...